docs/hosted-orchestration.md

```diff
@@ -0,0 +1,9 @@
+---
+id: hosted-orchestration
+title: Sauce Labs Hosted Orchestration
+sidebar_label: Getting Started
+---
+
+Execute tests up to 70% faster using Hosted Orchestration. Hosted Orchestration runs mobile and broswer tests for you from within the Sauce Labs grid. It works using our container execution environment. All you need to do is package your test code as a Docker image and grant us access. 
```

Comment:
Do we want to specify that it's Selenium and Appium? Saying "mobile" might give the false hope that we allow XCTest and Espresso

Reply:
hmm I would actually rather leave it more generic since this is really just about migrating where you run tests. For example, we're working on a Cypress example as well that basically takes the SauceCTL cypress docker image and runs from within hosted execution (without the need for a VM)